test/testGetMetadata.js

```diff
@@ -29,7 +29,7 @@ function callback(body) {
   parseFront(dat);
   split(dat, { logger });
   getmetadata(dat, { logger });
-  return dat.content.sections;
+  return dat.content.mdast;
```

Comment:
Expose the `mdast` node directly since most tests have a single section that will be unwrapped to it